Cart/Cart.Application/Interfaces/ICartService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Cart.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Cart.Application.Interfaces
+{
+    public interface ICartService
+    {
+        void Create(CartList createCart);
```

Comment:
In Such Cases a boolean or a string could have been returned as an acknowledgement